Tests/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_subdirectory(Packet++Test)
```

Comment:
I noticed that when building with Visual Studio there is a solution files (`.sln`) for:
- Everything - libs, tests, examples, etc.
- Just the examples
- Just the tutorials
- A few 3td party libs

However there are no solution files for:
- The tests
- Just the lib projects (Common++, Packet++, Pcap++)

![image](https://user-images.githubusercontent.com/9059541/187603954-edfe180a-1e70-4fff-95a0-c5978443a869.png)



The reason is that CMake generates a `.sln` file anywhere it finds a `project` command as explained here: https://cognitivewaves.wordpress.com/cmake-and-visual-studio/

What I think we can do:
- Add a `project` command in this file so it will generate a `.sln` file for the tests
- Not sure how to generate a `.sln` file for Common++, Packet++, Pcap++ 🤔